Apps/Playground/iOS/ViewController.swift

```diff
@@ -26,22 +30,24 @@ class ViewController: UIViewController {
 
             let rawMetalLayerPtr: UnsafeMutableRawPointer = Unmanaged.passUnretained(mtkView.layer).toOpaque()
 
-            let screenBounds = UIScreen.main.bounds
-            let width = screenBounds.size.width
-            let height = screenBounds.size.height
+            let width = view.bounds.size.width
+            let height = view.bounds.size.height
 
             appDelegate!._bridge!.init(rawMetalLayerPtr, width:Int32(width), height:Int32(height))
         }
     }
 
     override func viewWillLayoutSubviews() {
+        super.viewWillLayoutSubviews()
         let appDelegate = UIApplication.shared.delegate as? AppDelegate
+        if mtkView == nil {
+            return
+        }
         if appDelegate != nil {
-            let screenBounds = UIScreen.main.bounds
-            let width = screenBounds.size.width
-            let height = screenBounds.size.height
+            let width = view.bounds.size.width
+            let height = view.bounds.size.height
 
-            appDelegate!._bridge!.resize(Int32(width), height: Int32(height))
+            appDelegate!._bridge!.resize(Int32(width + 1), height: Int32(height + 1))
```

Comment:
i'm assuming there is a reason for the `+ 1`. maybe add a comment?

Reply:
I'll spend more time on this one. I don't really understand some of the results I get.

Reply:
Confirmed to be working on device & sim.